src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -364,16 +334,16 @@ private static void downloadJavaFXZip(String osarch, Path substratePath, Project
      *                    The contents of this zip file will be installed in graalvm/libs/clibraries/foo
      * @throws IOException
      */
-    public static void downloadZip(String urlZip, Path destination) throws IOException {
+    public void downloadZip(String urlZip, Path destination) throws IOException {
         String zip = destination.toAbsolutePath().toString()+".zip";
         Path zipPath = Paths.get(zip);
-        processZip(urlZip, zipPath, destination.getFileName().toString(), null, null);
+        processZip(urlZip, zipPath, destination.getFileName().toString(), null);
     }
 
-    private static void processZip(String urlZip, Path zipPath, String folder, String version, ProjectConfiguration configuration) throws IOException {
+    private void processZip(String urlZip, Path zipPath, String folder, String version) throws IOException {
```

Comment:
add docs about the parameters

Reply:
Will leave this alone for now... I think this method can be refactored. Once it's done - will add the docs